insights/parsers/krb5kdc_log.py

```diff
@@ -52,29 +52,20 @@ def get(self, keyword):
         * `pid`
         * `level`
         * `message`
-        * `line` - the full line as originally given.
+        * `raw_message` - the full line as originally given.
 
         If the log line is not in the standard format, only the 'line' field
         will be stored in the dictionary.
-
-        Parameters:
-            keyword(str): a keyword to search for in the line
-
-        Yields:
-            (dict): A dictionary for each parsed line in the input.
         '''
+
         line_re = re.compile(
             r'(?P<timestamp>\w{3} \d\d \d\d:\d\d:\d\d) ' +
             r'(?P<system>\w\S+) ' +
             r'(?P<service>\w+)\[(?P<pid>\d+)\]\((?P<level>\w+)\): ' +
             r'(?P<message>.*)'
         )
```

Comment:
Are there any necessaries for putting this `line_re` as a class constant value ?
We have examples at class ServerXMLRPCLog in parser rhn_logs.py.